web/client-api/src/main/java/io/deephaven/web/client/api/widget/JsWidget.java

```diff
@@ -189,6 +191,30 @@ public Promise<JsWidget> refetch() {
         });
     }
 
+    /**
+     * Exports another copy of this widget.
+     * 
+     * @return Promise resolving to a re-exported copy of this widget
+     */
+    @JsMethod
+    public Promise<JsWidget> copy() {
```

Comment:
I'd be inclined to call it "reexport()", especially if we're going to expose it as part of the public API - no copying is taking place on the server, it is just a new export of the exact same instance. That does imply a new objecttype plugin instance - one per _connection_, not per export - hard to say how much we want those lines blurred.

If it isn't exported, I'm less worried about the name.

Reply:
Changed to reexport and also remember the `@JsMethod`. I figure we can expose it in the future if we have a more general need for it.